createCard/action.yml

```diff
@@ -19,6 +19,12 @@ inputs:
   typeId:
     description: Optionally specify a card type id to use. Default card type will be used when not set.
     required: false
+  customId:
+    description: Optionally specify header for the new card.
+    required: false
+  externalLink:
+    description: Optionally specify a link to the GH issue.
```

Comment:
I was wondering whether to make it so that you have to provide a link and a label. That might need a little more thought.
Or, as you say, just make it non specific.

Reply:
I think collecting a label would be a good idea, so it can be used for many purposes.

Reply:
Done